terraform/environments/sprinkler/notifications.tf

```diff
@@ -0,0 +1,57 @@
+# Data source to get the ARN of an existing SNS topic
+data "aws_sns_topic" "existing_topic" {
+  name = "backup_failure_topic"
+}
+
+# Create an email subscription to the existing SNS topic
```

Comment:
You can set it up at first to test it, but I think it could be spammy to have it there permanently (but see what others think, I did not set up an email notification for the instance scheduler and all other cloudwatch alerts are also not set up with an email notification)

Reply:
Good point, I have it in there just to test the creation but going forward I could easily remove it!